### ISSUE TITLE
Protection Force bug

### ISSUE THREAD

**Issue by orderorder** [shinyquagsire23/OpenJKDF2#331]
If i save the game using the protection force, when I load The game the force shield gets stuck in the position where I saved the game. It dont moves with the player

![Sin nombre](https://github.com/shinyquagsire23/OpenJKDF2/assets/7293829/4b2d92ff-f1cb-48ed-9ff2-c734e10e341f)

**Comment by shinyquagsire23** [comment]
Hmm can't seem to reproduce this one, does it still happen on your end?

**Comment by orderorder** [comment]
> Hmm can't seem to reproduce this one, does it still happen on your end?

Hi, I lost my savegame so I had to use debug to select the last level and I had to use cheats. It seems that the bug has been fixed.
I saw that there is a fork that implements new features https://github.com/LadyEebs/OpenJKDF2 but it doesn't compile on Linux so I can't test it, I'm telling you in case you find it interesting.
Another question, is there a way to use the mouse wheel to change weapons? Thanks
